pkg/apis/devices/v1alpha2/doc.go

```diff
@@ -13,6 +13,6 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-
```

Comment:
I think this space line should not be deleted. Keep a space line between copyright and context. Same with others doc.go